drain-cleaner/modules/drain-cleaner/drain_cleaner_run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+set +x
+
+# Exit when we run out of heap memory
+JAVA_OPTS="${JAVA_OPTS} -XX:+ExitOnOutOfMemoryError"
+
+# Disable FIPS if needed
+if [ "$FIPS_MODE" = "disabled" ]; then
+    export JAVA_OPTS="${JAVA_OPTS} -Dcom.redhat.fips=false"
+fi
+
+# Start Drain Cleaner
+#exec /usr/bin/tini -s -w -e 143 -- java $JAVA_OPTS -classpath "$JAVA_CLASSPATH" "$JAVA_MAIN" "$@"
+exec /usr/bin/tini -s -w -e 143 -- java $JAVA_OPTS -jar /opt/strimzi/drain-cleaner.jar "$@"
```

Comment:
What is the name of the jar we use upstream? From what I understand the name changed, if that is the case I would be agree that we can change the name of the jar here and in the modiles file to match the name used upstream. One less difference between upstream and downstream run scripts

Reply:
In upstream we have `/opt/strimzi/quarkus-run.jar` and yes it would be good if we can align